src/Configuration/MigrationConfigurationCollection.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+
+namespace Quidco\DbSampler\Configuration;
+
+class MigrationConfigurationCollection
+{
+    /**
+     * @var array
+     */
+    private $configs;
+
+    private function __construct(array $configs)
+    {
+        $this->configs = $configs;
+    }
+
+    public static function fromFilePaths(array $filepaths): self
+    {
+        $configs = [];
+
+        foreach ($filepaths as $migrationFilePath) {
+            try {
+                if (is_dir($migrationFilePath)) {
```

Comment:
Worth making this change

```suggestion
                $migrationFiles = [$migrationFilePath];

                if (is_dir($migrationFilePath)) {
                    $migrationFiles = glob(rtrim($migrationFilePath, '/') . '/*.json');
                }
```

Reply:
Yeah, i think you're right, cheers. A lot of the contents of these files have just been a cut & paste refactoring of the original `App`, `ConsoleRunner`, and `Migrator` classes in order to make future changes simpler.

Reply:
Suggestion applied in b5003ef